### PR TITLE
MCS-220

### PR DESCRIPTION
**Pull request by jaudick**
I never actually touched QualitySettings.asset and DynamicsManager.asset. If someone could tell me what going on with those I would appreciate it.
I also added the prefab to MCS-Private

### CODE REVIEW

File: unity/ProjectSettings/DynamicsManager.asset [hunk 1]
```diff
@@ -18,7 +18,7 @@ PhysicsManager:
   m_ClothInterCollisionStiffness: 0
   m_ContactsGeneration: 1
   m_LayerCollisionMatrix: c9f7ffffc8f0ffffc8f0ffffffffffffc8f0ffffc8f0ffffffffffffffffffffc9f7ffffc9f7ffffc9f3ffffc8f0ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
-  m_AutoSimulation: 1
+  m_AutoSimulation: 0
```
Comment (brianpippin) [inline unity/ProjectSettings/DynamicsManager.asset:21]: Should this have changed?

File: unity/ProjectSettings/QualitySettings.asset [hunk 2]
```diff
@@ -287,54 +287,4 @@ QualitySettings:
     asyncUploadPersistentBuffer: 1
     resolutionScalingFixedDPIFactor: 1
     excludedTargetPlatforms: []
-  - serializedVersion: 2
```
Comment (brianpippin) [inline unity/ProjectSettings/QualitySettings.asset:290]: Should we be removing these? @ThomasSchellenbergNextCentury

File: unity/ProjectSettings/DynamicsManager.asset (hunk 1 above)
Comment (jaudick) [reply unity/ProjectSettings/DynamicsManager.asset:21]: No, I don't know how this happened I never opened this file

File: unity/ProjectSettings/QualitySettings.asset (hunk 2 above)
Comment (jaudick) [reply unity/ProjectSettings/QualitySettings.asset:290]: I never opened this either, I'll see what I can do

Comment (ThomasSchellenbergNextCentury) [reply unity/ProjectSettings/QualitySettings.asset:290]: Yeah, I'm not sure what this file does.  Maybe you changed an option in your Editor?  Regardless, you should be able to find some help online on how to use git to "checkout" a specific file from a specific commit.

Comment (jaudick) [reply unity/ProjectSettings/QualitySettings.asset]: This should be fixed now

File: unity/ProjectSettings/DynamicsManager.asset (hunk 1 above)
Comment (jaudick) [reply unity/ProjectSettings/DynamicsManager.asset]: @brianpippin this should be fixed now

File: unity/ProjectSettings/QualitySettings.asset (hunk 2 above)
Comment (jaudick) [reply unity/ProjectSettings/QualitySettings.asset]: @ThomasSchellenbergNextCentury  this should be fixed now

**Pull request merged**